OWNERS.md

```diff
@@ -0,0 +1,54 @@
+### Core Maintainers
+
+Ria Bhatia (ribhatia@microsoft.com)
+
+Robbie Zhang (junjiez@microsoft.com)
```

Comment:
Should we remove Robbie?

Reply:
I am OK with keeping him for another three months, since only now will the project be introducing the governing rules. But if you, as Robbie co-workers, have insider data that points to Robbie no longer be involved, I'm also OK with removing.